__tests__/task-requests/task-requestDetails.test.js

```diff
@@ -46,8 +46,9 @@ describe('Request container for non-super users', () => {
 
   it('Approve and Reject buttons should not render for non-super users', async function () {
     await page.goto(
-      'http://localhost:8000/task-requests/details/?id=dM5wwD9QsiTzi7eG7Oq5&&dev=true',
+      'http://localhost:8000/task-requests/details/?id=dM5wwD9QsiTzi7eG7Oq5&dev=true',
     );
+    await page.waitForTimeout(2000);
```

Comment:
So here we are waiting for 2s, and then checking rest of the things right?
So won't this increase test run time, and now its been added at 4 places, which increases it by 8s?

Reply:
I can replace the page.waitForTimeout(2000) with page.waitForSelector to make the tests faster and avoid fixed delays, especially since it's used in multiple places. Let me know if you'd like me to go ahead with this adjustment.

Reply:
Also when the feature flag will be removed this whole delay can also be removed